### PR TITLE
fix(assets/webidl.css): fix color contrast in IDL header

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: assets/respec.css [hunk 1]
```diff
@@ -79,7 +79,7 @@ cite .bibref {
 }
 
 code {
-  color: #c83500;
+  color: #c63501;
```
Comment (sidvishnoi) [inline assets/respec.css]: - `#c83500` (or `hsl(15.9, 100%, 39.2%)`) (color contrast: 4.48, fail)
- `#c63501` (or `hsl(15.8, 99%, 39.0%)`) (color contrast: 4.54, pass)
(at least the a11y linter is not labeling 20% of page as error)

File: assets/webidl.css [hunk 2]
```diff
@@ -24,7 +24,6 @@ pre.idl {
 }
 
 .idlHeader a.self-link {
-   color: #fff;
```
Comment (sidvishnoi) [inline assets/webidl.css]: It will use the default dark blue-ish link color from base.css

File: assets/webidl.css [hunk 3]
```diff
@@ -14,7 +14,7 @@ pre.idl {
 .idlHeader {
   display: block;
   width: 150px;
-  background: #90b8de;
+  background: #8ccbf2;
```
Comment (sidvishnoi) [inline assets/webidl.css]: Same as border color or IDL block

Comment (sidvishnoi) [reply assets/webidl.css]: Before:
image

After:
image

Comment (marcoscaceres) [reply assets/webidl.css]: hehe, 👓... totally see the difference!

Comment (marcoscaceres) [reply assets/webidl.css]: in all seriousness, I did prefer the white... but can live with the dark.

**Pull request merged**